src/main/java/seedu/duke/data/Budget.java

```diff
@@ -1,4 +1,33 @@
 package seedu.duke.data;
 
 public class Budget {
+    private double amount;
+
+    public Budget(double amount) {
+        if (amount > 5000) {
+            this.amount = 5000;
+        } else if (amount < 0) {
+            this.amount = 0;
+        } else {
+            this.amount = amount;
+        }
+    }
+
+    public void setBudget(double amount) {
+        if (amount > 5000) {
+            this.amount = 5000;
+        } else if (amount < 0) {
+            this.amount = 0;
+        } else {
+            this.amount = amount;
+        }
+    }
```

Comment:
Hello here's my suggestion for the three if-else cases :)

1. define max and min values in your budget class
final static double MAX = 5000;
final static double MIN = 0;

2. The empty constructor initializes budget to zero
public Budget(){
    this.amount = 0;
  }

3. set budget relies on min/max value (handles negative values too!) 🥂 
 public void setBudget(double amount){
    if(amount > 5000){
   this.amount = Math.min(amount,MAX);
    }else{
   this.amount = Math.max(amount,0);
    }
  }

hope this helps! 💃

Reply:
Good idea, thanks!!

Reply:
Actually, for 3, the if/else statements can just be removed entirely